azure/template.json

```diff
@@ -284,7 +291,20 @@
     "databaseResourceNamePrefix": "[toLower(concat(parameters('resourceEnvironmentName'),'-', parameters('databaseServiceName')))]",
     "databaseServerName": "[concat(variables('databaseResourceNamePrefix'), '-psql')]",
     "databaseUsername": "[concat(parameters('databaseUsername'), '@', variables('databaseServerName'))]",
-    "storageAccountName": "[replace(concat(variables('resourceNamePrefix'), 'str'), '-', '')]"
+    "storageAccountName": "[replace(concat(variables('resourceNamePrefix'), 'str'), '-', '')]",
+    "defaultAvailabilityCheckHosts": "[if(greater(length(parameters('customHostName')), 0), createArray(concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping'), concat('check:', parameters('customHostName'), '/ping')), createArray(concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping')))]",
```

Comment:
Should this be:

```suggestion
    "defaultAvailabilityCheckHosts": "[if(greater(length(parameters('customHostName')), 0), createArray(concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping'), concat('azcheck:', parameters('customHostName'), '/ping')), createArray(concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping')))]",
```

Reply:
I found this hard to read, so I've reformatted it. Nothing visibly wrong, it's just way easier to read. :)

```C
if(greater(length(parameters('customHostName')), 0),
   createArray(
     concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping'),
     concat('check:', parameters('customHostName'), '/ping')
   ),
   createArray(
     concat('azcheck:', variables('appServiceName'), '.azurewebsites.net/ping')
   )
)
```

Reply:
> Should this be:

There are two availability checks, `check`  and `azcheck`.  One for azurewebsite.net address and other for custom hostname.